content/tr/index.md

```diff
@@ -19,4 +19,4 @@ or
 
 [Nightly releases](https://iojs.org/download/nightly/) are available for testing.
 
-[Frequently Asked Questions](/faq.html)
\ No newline at end of file
+[Frequently Asked Questions](/faq.html)
```

Comment:
The following settings can be recommended to contributors using these editors:

Atom: `Packages > Core > Whitespace > Ensure Single Trailing Newline`
Sublime: `"ensure_newline_at_eof_on_save": true`